### ISSUE TITLE
rename MetaCPAN::API

### ISSUE THREAD

**Issue by haarg** [metacpan/metacpan-api#874]
MetaCPAN::API is a poor choice for a package name, since there is already a module with that name on CPAN.  This is confusing and could lead to weird issues if we ever pulled in a dependency that was still using MetaCPAN::API.

**Comment by jberger** [comment]
It was chosen to be symmetric with ::Web. I see those modules are yours and are deprecated. Is there any chance we could just remove them? Otherwise, what came do you propose?

**Comment by haarg** [comment]
I took over maintenance of MetaCPAN::API to do fixes allow it to continue working with the v1 API.  It has enough code dependent on it that I have no intention of removing it.

Some names without much thought put into them:
MetaCPAN::APIServer
MetaCPAN::Server::API (and maybe even moving web to MetaCPAN::Server::Web)
metacpan::api

**Comment by oalders** [comment]
I like `MetaCPAN::Server::API` or even `MetaCPAN::Service::API`.

**Comment by ssoriche** [comment]
I like `MetaCPAN::APIServer` as it's pretty explicit as to what it is, without colliding with the existing Catalyst implementation.

**Comment by mohawk2** [comment]
Was this discussed / resolved at PTS 2019?

**Comment by oalders** [comment]
No, I don't think there's any resolution on this. `MetaCPAN::APIServer` seems like the most succinct of the suggestions.

**Comment by mohawk2** [comment]
You're quite right it's succinct. It does read a bit wrong to me, just because I'd prefer to not combine two "words" (API and server) without a `::` between. However, it's not my call :-)

**Comment by szabgab** [comment]
Maybe MetaCPAN::Public::API
